phys/module_ra_cam.F

```diff
@@ -907,7 +908,8 @@ subroutine camrad(RTHRATENLW,RTHRATENSW,                           &
         SWNIRDIF(I,J) = solld(ii) !SSiB 
       enddo
       endif
-  CASE (CLMSCHEME)
+
```

Comment:
as far as I can tell, no change is needed in module_ra_cam.F

Reply:
Thanks for the comment. 
As far as I can tell, no change has been done for this file. 
It seems like @billsacks previously added some todos here which is removed in 83dbf7b6, but caused the extra line we see here. I actually revert this back to be identical to before. So basically, this file should not even show up in the files changed list.